src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -448,10 +448,12 @@ export default class CanvasController {
      * @method canvas_update_post
      */
     canvas_update_post() {
-        const elapsed = GameController.game_time() - AirportController.airport_get().start;
+        const airport = AirportController.airport_get();
+        const elapsed = GameController.game_time() - airport.start;
         const alpha = extrapolate_range_clamp(0.1, elapsed, 0.4, 0, 1);
         const framestep = Math.round(extrapolate_range_clamp(1, GameController.game.speedup, 10, 30, 1));
         const shouldUpdate = !GameController.game_paused() && TimeKeeper.frames % framestep === 0;
+        const shouldUpdateWind = !GameController.game_paused() && TimeKeeper.frames % 50 === 1;
```

Comment:
this method is concerned with drawing the current state of all the drawable game objects.  currently, this method is gross.  like, really gross.  we are doing so much here that we shouldn't be doing here.

the updating of the wind should happen elsewhere completely outside of this class so that once it comes time to draw it we are just grabbing values that already exist.  Any math or calculations that happen slow down the drawing process and lead to dropped frames (currently a big problem with this method)